setup.py

```diff
@@ -13,7 +13,7 @@
 	packages = ["bfx_qc_reporter", "bfx_qc_reporter.util"],
 	package_dir = {"bfx_qc_reporter" : "src/python/bfx_qc_reporter", "bfx_qc_reporter.util" : "src/python/bfx_qc_reporter/util"},
 	#package_data = {"bfx_qc_reporter" : "resources/*.csv"},
-	package_data = {"bfx_qc_reporter" : "src/python/bfx_qc_reporter/resources/*.csv"},
+        package_data = {"bfx_qc_reporter" : ["src/python/bfx_qc_reporter/resources/*.csv"]},
```

Comment:
can you fix the silly indenting here and above?  You can also remove the commented out lines.